src/contactForm.js

```diff
@@ -22,8 +22,8 @@ class RecrasContactForm {
             this.languageHelper.setLocale(this.options.getLocale());
         }
 
-        this.fetchJson = url => RecrasHttpHelper.fetchJson(url, this.error);
-        this.postJson = (url, data) => RecrasHttpHelper.postJson(this.options.getApiBase() + url, data, this.error);
+        this.fetchJson = url => RecrasHttpHelper.fetchJson(url, this.error.bind(this));
+        this.postJson = (url, data) => RecrasHttpHelper.postJson(this.options.getApiBase() + url, data, this.error.bind(this));
```

Comment:
are there more uses of `this.error` that need a `.bind(this)` to work, or are these all the occurences?

Reply:
There are no other occurrences that need `bind`